Add 2022 Keynote Nomination Post (#325)
SeaGL/seagl.github.io: _posts/2022-04-05-keynote-nomination.md

---
layout: post
title: 'Now Accepting Nominations for SeaGL 2022 Keynotes'
status: publish
type: post
published: true
categories: news
tags: '2022'
---

> One of the things that makes SeaGL unique from other conferences is that we allow the public to play a role in our keynote selection process. For the next two weeks, we will be accepting nominations for people you would like to see present this year's keynote talks.

Reviewer: "we allow the public to play a role" sounds a little...IDK. Words. Maybe instead something like "our community helps us select our keynote speakers." Focus more on participation and not permission, if that makes sense.

> While you are welcome to self-nominate yourself, we encourage you to take some time to think about other people you have seen present in the past who you think have an important message to share and nominate them as well.

Reviewer: "self-nominate yourself" may be slightly redundant. :-)

Author: ```suggestion
While you are welcome to nominate yourself, we encourage you to take some time to think about other people you have seen present in the past who you think have an important message to share and nominate them as well.
```

> To nominate someone, please use our [keynote speaker nomination form](https://docs.google.com/forms/d/e/1FAIpQLSfvBo7SX6ku6vmLLvtzbJ_c80b_WfvhUl-GAXpc6nssXCvg-w/viewform?usp=sf_link). You can submit as many nominations as you want (the more the merrier!), and all submissions are copmletely anonymous. **The form will be open until Friday, April 22 at 11:59pm PDT.**

Author: ```suggestion
To nominate someone, please use our [keynote speaker nomination form](https://docs.google.com/forms/d/e/1FAIpQLSfvBo7SX6ku6vmLLvtzbJ_c80b_WfvhUl-GAXpc6nssXCvg-w/viewform?usp=sf_link). You can submit as many nominations as you want (the more the merrier!), and all submissions are completely anonymous. **The form will be open until Friday, April 28 at 11:59pm PDT.**
```

> Nominees need not have prior keynote experience; we are interested in both new and veteran keynote presenters. If they have been doing exciting work in the Open Source community and have and exciting story to share, we want to know about them! We also strive to have representation from a broad range of backgrounds.

Reviewer: I know we don't have a conference theme yet, but is there a theme or a...general sort of thing we'd like to publicly say we're focusing on. ("No" is a fine answer, but if we do, we should call it out)

Reviewer 2: Yeah ten... but nothing has quite felt right yet.

We challenge you to think about people who
  * You go out of your way to read all of their Open Source posts on social media
  * Might only have a small following, but are doing amazing work in their own little corner
  * Has done work to empower others and challenge social norms
  * Are always tinkering with new and exciting projects that inspire others to try new things
  * Are slowly, but steadily, working to make the Open Source community and the world a better place for everyone
and [nominate them](https://docs.google.com/forms/d/e/1FAIpQLSfvBo7SX6ku6vmLLvtzbJ_c80b_WfvhUl-GAXpc6nssXCvg-w/viewform?usp=sf_link) to be a SeaGL Keynote presenter!

> SeaGL will take place virtually November 4-5, 2022.

Reviewer: We are discussing the format as I type this.

Author: ```suggestion
SeaGL will take place November 4-5, 2022.
```
